usecases.md

```diff
@@ -3,3 +3,418 @@ layout: base
+
+1. Issue time (16:00)
+2. Number of intervals (3)
+3. Interval duration (1 hour)
+4. Interval label (left endpoint)
+5. Value unit (MW)
+6. Value type (average)
+
+**Forecast**. A _forecast_ is a serially complete time series of (Time, Value) pairs spanning from a s_tart time_ to an e_nd time_. A single forecast run is a forecast. A forecast can also comprise the concatenation of a series of identically specified forecast runs with different issue times, as shown in the figures below. For example, three forecast runs (green) could be:
+
+- {(13:00, 10MW)} issued at 12:00, and
+- {(14:00, 7MW)} issued at 13:00, and
+- {(15:00, 5MW)} issued at 14:00.
+
+From these three runs, the user could upload an hour ahead, hour interval forecast of:
+
+- {(13:00, 10MW), (14:00, 7MW), (15:00, 5MW)}
+
+![timeline_concat_1h](https://raw.githubusercontent.com/wholmgren/fxtimeline/c92cd946a877fdc67a9662fe28b5410d27d790cd/timeline_concat_1h.svg?sanitize=true)
+
+The framework also supports the concatenation of submissions with more than one interval, so long as the intervals do not overlap. The example below shows how three different hour ahead, 15 minute interval, hour duration forecast runs may be concatenated into a single forecast for evaluation.
+
+ ![timeline_concat](https://raw.githubusercontent.com/wholmgren/fxtimeline/c92cd946a877fdc67a9662fe28b5410d27d790cd/timeline_concat.svg?sanitize=true)
+
+The framework is primarily designed to assess a _forecast_. For most use cases (1A-1E), the framework expects forecast providers to upload a _forecast_. For the Forecast Trial use case (1F), the framework expects forecast providers to upload a series of _forecast runs_ that the framework can concatenate into a _forecast_ for evaluation.
+
+A stretch goal is to support the use case of analyzing multiple forecast runs with the same valid times (1G). For example, the user could upload each of the forecast runs specified above, and then use the framework to merge the forecast runs into a 0, 1, or 2 hour ahead forecast for evaluation. The figure below shows three forecasts runs (green) merged into two different evaluation forecasts (blue, red).
+
+ ![timeline_merge](https://raw.githubusercontent.com/wholmgren/fxtimeline/c92cd946a877fdc67a9662fe28b5410d27d790cd/timeline_merged.svg?sanitize=true)
+
+
+## Forecast parameters
+
+A _forecast_ is a piecewise-continuous time series of values parameterized by:
+
+1. Lead time to start of forecast
+2. Interval duration
+3. Intervals per submission
+4. Forecast issue frequency
+5. Value type (e.g. mean, max, 95th percentile, instantaneous)
```

Comment:
Aidan: no mention of start/end labeling here. might need example because it's something that normally needs clarification

Reply:
Added 6. Interval label (start or end)